### PR TITLE
Add tests for the new aggregated GeometryHash 

### LINKED ISSUE #16: Add tests for the new GeometryHash implementation
See https://github.com/BHoM/BHoM_Engine/issues/3251

### PR DESCRIPTION
**Pull request by alelom**
### Description 

Closes https://github.com/BHoM/DiffingTests_Prototypes/issues/16

Adds tests for https://github.com/BHoM/BHoM_Engine/pull/3257

### CODE REVIEW

**Comment by FraserGreenroyd** [comment]
@BHoMBot check copyright-compliance
@BHoMBot check project-compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot fix copyright headers ref. 20892859241

**Comment by FraserGreenroyd** [comment]
@BHoMBot check copyright-compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot fix copyright headers ref. 20892910901

File: DiffingTests/DiffingTests.csproj [hunk 1]
```diff
@@ -33,21 +37,9 @@
     <Reference Include="Analytical_oM">
       <HintPath>C:\ProgramData\BHoM\Assemblies\Analytical_oM.dll</HintPath>
```
Comment (FraserGreenroyd) [inline DiffingTests/DiffingTests.csproj]:
```suggestion
      <HintPath>$(ProgramData)\BHoM\Assemblies\Analytical_oM.dll</HintPath>
```

File: DiffingTests/DiffingTests.csproj [hunk 2]
```diff
@@ -33,21 +37,9 @@
...
-      <HintPath>C:\ProgramData\BHoM\Assemblies\BHoM.dll</HintPath>
-    </Reference>
-    <Reference Include="BHoM_Engine">
-      <HintPath>C:\ProgramData\BHoM\Assemblies\BHoM_Engine.dll</HintPath>
-    </Reference>
-    <Reference Include="Diffing_oM">
-      <HintPath>C:\ProgramData\BHoM\Assemblies\Diffing_oM.dll</HintPath>
-    </Reference>
     <Reference Include="Dimensional_oM">
       <HintPath>C:\ProgramData\BHoM\Assemblies\Dimensional_oM.dll</HintPath>
```
Comment (FraserGreenroyd) [inline DiffingTests/DiffingTests.csproj]:
```suggestion
      <HintPath>$(ProgramData)\BHoM\Assemblies\Dimensional_oM.dll</HintPath>
```

File: DiffingTests/DiffingTests.csproj [hunk 3]
```diff
@@ -33,21 +37,9 @@
...
     </Reference>
-    <Reference Include="Geometry_Engine">
-      <HintPath>C:\ProgramData\BHoM\Assemblies\Geometry_Engine.dll</HintPath>
-    </Reference>
     <Reference Include="Geometry_oM">
       <HintPath>C:\ProgramData\BHoM\Assemblies\Geometry_oM.dll</HintPath>
```
Comment (FraserGreenroyd) [inline DiffingTests/DiffingTests.csproj]:
```suggestion
      <HintPath>$(ProgramData)\BHoM\Assemblies\Geometry_oM.dll</HintPath>
```

**Comment by alelom** [comment]
check project-compliance

**Comment by alelom** [comment]
@BHoMBot check project-compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a DevOps instruction. I am authorising dispensation to be granted on check ref. 20893103183

**Comment by FraserGreenroyd** [comment]
@BHoMBot check copyright-compliance

**Pull request merged**